docs/user-guide/ze-working-with-jobs.md

```diff
@@ -0,0 +1,115 @@
+# Working with jobs
+
+### Viewing a job
+
+1. Expand **JOBS** in the **Side Bar**.
+2. Open a directory with JCL files.
+3. Right-click on the JCL file you want to view, and select the **Get JCL** option.
+
+   ![View JOB](../images/ze/ZE-jobs-get-jcl.gif)
+
+## Downloading spool content
+
+1. Expand **JOBS** in the **Side Bar**.
+2. Open a directory with JCL files.
+3. Click the **Download** icon next to a folder with the spool content.
+4. Save the file on your computer.
+
+   ![Download Spool](../images/ze/ZE-jobs-download-spool.gif)
+
+## Issuing MVS commands
+
+1. Expand **JOBS** in the **Side Bar**.
+2. Right-click on your profile and select the **Issue MVS Command** option.
+
+   Alternatively, press the `F1` key to open the **Command Pallette**, then select the **Zowe Expolorer: Issue MVS Command** option.
```

Comment:
Alternatively, press the `F1` key to open the **Command Pallette**, and then select the **Zowe Expolorer: Issue MVS Command** option.